types/entropy__winston-elasticsearch-apm/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {}
```

Comment:
you should be able to remove this field entirely 😉

Reply:
@weswigham done